torch/testing/_internal/common_methods_invocations.py

```diff
@@ -666,6 +668,95 @@ def sample_inputs_linalg_norm(op_info, device, dtype, requires_grad):
                             dim=(0, 1))))
         return inputs
 
+
+def sample_inputs_norm(op_info, device, dtype, requires_grad, sample_types='default', **kwargs):
+    make_arg = partial(make_tensor, device=device, dtype=dtype, requires_grad=requires_grad)
+
+    cases_nuc = (
```

Comment:
This might be more readable if there was just one sample input func per norm variant.

Another option would be to try and create a more generic generation for norm inputs. That seems tricky, however, and ultimately we plan to be rid of torch.norm, so extending its test coverage isn't especially interesting.

Reply:
Sure! Will split the sample func.